app/controllers/api_controller.rb

```diff
@@ -17,6 +18,10 @@ def current_user
     doorkeeper_token&.application
   end
 
+  def user_for_paper_trail
+    current_user.owner_id
+  end
+
```

Comment:
:+1: it looks like this will extend nicely once we have the user information from the frontend :100: